ocaml/client_records/records.ml

```diff
@@ -1098,6 +1098,8 @@ let host_record rpc session_id host =
 				~get:(fun () -> String.concat "; " (List.map Int64.to_string (x ()).API.host_virtual_hardware_platform_versions)) 
 				~get_set:(fun () -> List.map Int64.to_string (x ()).API.host_virtual_hardware_platform_versions) ();
 			make_field ~name:"control-domain-uuid" ~get:(fun () -> get_uuid_from_ref (x ()).API.host_control_domain) ();
+			make_field ~name:"resident-vms"
+				~get:(fun () -> String.concat "; " (List.map get_uuid_from_ref (x ()).API.host_resident_VMs)) ~expensive:true ();
```

Comment:
@johnelse,  it seems I don't have a proper understanding of the meaning of `expensive`; can you point me to relevant docs?

@robhoes, host.resident_VMs is DynamicRO, why does it need a ~get_set?